src/request/components/RequestDetailPanelData.tsx

```diff
@@ -0,0 +1,48 @@
+import { MasterDetailTriPanel } from './MasterDetailTriPanel';
+import { RequestDetailPanelDataOperationCommandContainer } from '../containers/RequestDetailPanelDataOperationCommandContainer';
+import { RequestDetailPanelDataOperationTableContainer } from '../containers/RequestDetailPanelDataOperationTableContainer';
+
+import * as React from 'react';
+import Highlight = require('react-highlight');
+
+export interface IDataComponentProps {
+    totalOperationCount: number;
+}
+
+export class DataComponent extends React.Component<IDataComponentProps, {}> {
+    public render() {
+        return (
+            <div className='tab-data'>
+            {
+                this.props.totalOperationCount > 0
+                    ? <MasterDetailTriPanel masterPanel={this.renderMaster()} leftDetailPanel={this.renderLeftDetail()} leftDetailPanelTitle='Command' rightDetailPanel={this.renderRightDetail()} rightDetailPanelTitle='Summary' />
+                    : <div className='tab-section text-minor'>Could not find any data.</div>  
+            }
+            </div>
+        );
+    }
+    
+    private renderMaster() {
+        return (
+            <div className='tab-data-master'>
+                <div className='tab-data-operation-count'>{this.getTotalOperationCountText()}</div>
+                <br />
+                <RequestDetailPanelDataOperationTableContainer />
+            </div>
+        );
+    }
+    
+    private renderLeftDetail() {
+        return (
+            <RequestDetailPanelDataOperationCommandContainer />
+        )
+    }
+    
+    private renderRightDetail() {
+        return null;
+    }
+    
+    private getTotalOperationCountText() {
+        return this.props.totalOperationCount + (this.props.totalOperationCount === 1 ? ' Operation' : ' Operations');
+    }
```

Comment:
Yeah, pluralization tends to be highly-contextual, meaning that automating that in a service is difficult.  (As soon as you get into languages that have gender-ized articles, like German, the entire  phrase has to be localized, not just individual words.)  You pretty much end up with the same code that's here--a test for 0/1/>1 and then the resource strings that represent each case.  You might be able to abstract away the test, but you still have to pass the resource string names for each case.